index.html

```diff
@@ -1,12 +1,122 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Нёрдс</title>
-  </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
-
-  </body>
-</html>
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>NЁRDS</title>
+</head>
+<body>
+	<header>
+		<nav>
+			<a>
+				<img src="https://via.placeholder.com/150x100" alt="">
+			</a>
+			<ul>
+				<li><a href="studio.html">Студия</a></li>
+				<li><a href="clients.html">Клиенты</a></li>
+				<li><a href="catalog.html">Магазин</a></li>
+				<li><a href="contacts.html">Контакты</a></li>
+			</ul>
+			<div>
+				<a href="blank.html"><img src="https://via.placeholder.com/150x100" alt=""></a>
+				<a href="blank.html">Корзина</a>
+			</div>	
+		</nav>
+	</header>
+	<main>
+		<h1>Главная страница Nerds</h1>
+		<section>
+			<h2>Преимущества</h2>
+			<h3>Долго, дорого, скурпулёзно.</h3>
+			<p>Математически выверенный дизайн<br> для вашего сайта или мобильного приложения</p>
+			<a href="#">Узнать больше</a>
+			<img src="https://via.placeholder.com/150x100" alt="">
+		</section>
+		<section>
+			<h2>Услуги</h2>
+			<ul>
+				<li>
+					<img src="https://via.placeholder.com/150x100" alt="">
+					<h3>Веб-сайты</h3>
+					<button type="button">Заказать</button>
+				</li>
+				<li>
+					<img src="https://via.placeholder.com/150x100" alt="">
+					<h3>Приложения</h3>
+					<button type="button">Заказать</button>
+				</li>
+				<li>
+					<img src="https://via.placeholder.com/150x100" alt="">
+					<h3>Презинтации</h3>
```

Comment:
```suggestion
					<h3>Презентации</h3>
```

Текст лучше копировать из макета, так меньше вероятность допустить орфографические ошибки (но бывают и в макетах ошибки 😏 ).